subprojects/docs/src/docs/release/release-features.txt

```diff
@@ -0,0 +1,2 @@
+ - Feature variants, aka optional dependencies
+ - Type-safe accessors in pre-compiled script plugins
```

Comment:
```suggestion
 - Type-safe accessors in Kotlin pre-compiled script plugins
```

Groovy pre-compiled script plugins aren't a thing, [yet](https://github.com/gradle/gradle/issues/8610).

Reply:
Thanks @eskatos. I included this word in a different commit.